backend/graphql/index.ts

```diff
@@ -64,9 +64,7 @@ const graphQLMiddlewares = {
     simpleEntity: authorizedByAllRoles(),
     simpleEntities: authorizedByAllRoles(),
     userById: authorizedByAdmin(),
-    userByEmail: authorizedByAdmin(),
```

Comment:
used in `FirebaseAction` to get role of user (based on email) -> to determine which flows to use